pkg/fanal/handler/misconf/misconf_test.go

```diff
@@ -80,44 +87,158 @@ func Test_FindingFSTarget(t *testing.T) {
+			},
+			wantTarget: string(os.PathSeparator),
 			wantPaths:  []string{"foo", "bar"},
 		},
 		{
-			input:      []string{"/", "/bar"},
-			wantTarget: "/",
+			input:      []string{string(os.PathSeparator), filepath.Join(string(os.PathSeparator), "bar")},
+			wantTarget: string(os.PathSeparator),
 			wantPaths:  []string{".", "bar"},
 		},
 	}
 
 	for _, test := range tests {
 		t.Run(fmt.Sprintf("%#v", test.input), func(t *testing.T) {
+
+			if runtime.GOOS == "windows" {
+				cwd, err := os.Getwd()
+				require.NoError(t, err)
+				vol := filepath.VolumeName(cwd)
+
+				if test.wantTarget != "" {
+					test.wantTarget = filepath.Join(vol, string(os.PathSeparator), test.wantTarget)
+				} else {
+					test.wantTarget = vol
+				}
+			}
+
+			target, paths, err := findFSTarget(test.input)
+			if test.wantErr {
+				require.Error(t, err)
+			} else {
+				assert.Equal(t, test.wantTarget, target)
+				assert.Equal(t, test.wantPaths, paths)
+			}
+		})
+	}
+
+}
+
+func Test_FindingFSTarget_Windows(t *testing.T) {
+	if runtime.GOOS != "windows" {
+		t.Skip("test only runs on Windows - covered by Test_FindingFSTarget")
+	}
+
+	tests := []struct {
+		input      []string
+		wantTarget string
+		wantPaths  []string
+		wantErr    bool
+	}{
+		{
+			input:   nil,
+			wantErr: true,
+		},
+		{
+			input:      []string{string(os.PathSeparator)},
+			wantTarget: string(os.PathSeparator),
+			wantPaths:  []string{"."},
+		},
+		{
+			input:      []string{filepath.Join(string(os.PathSeparator), "home", "user")},
+			wantTarget: filepath.Join("home", "user"),
+			wantPaths:  []string{"."},
+		},
+		{
+			input: []string{
+				filepath.Join(string(os.PathSeparator), "home", "user"),
+				filepath.Join(string(os.PathSeparator), "home", "user", "something"),
+			},
+			wantTarget: filepath.Join("home", "user"),
+			wantPaths:  []string{".", "/something"},
+		},
+		{
+			input: []string{
+				filepath.Join(string(os.PathSeparator), "home", "user"),
+				filepath.Join(string(os.PathSeparator), "home", "user", "something", "else"),
+			},
+			wantTarget: filepath.Join("home", "user"),
+			wantPaths:  []string{".", "/something/else"},
+		},
+		{
+			input: []string{
+				filepath.Join(string(os.PathSeparator), "home", "user"),
+				filepath.Join(string(os.PathSeparator), "home", "user2", "something", "else"),
+			},
+			wantTarget: filepath.Join("home"),
+			wantPaths:  []string{"/user", "/user2/something/else"},
+		},
+		{
+			input: []string{
+				filepath.Join(string(os.PathSeparator), "foo"), filepath.Join(string(os.PathSeparator), "bar"),
+			},
+			wantTarget: "",
+			wantPaths:  []string{"/foo", "/bar"},
+		},
+		{
+			input:      []string{string(os.PathSeparator), filepath.Join(string(os.PathSeparator), "bar")},
+			wantTarget: "",
+			wantPaths:  []string{"/", "/bar"},
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(fmt.Sprintf("%#v", test.input), func(t *testing.T) {
+
+			if runtime.GOOS == "windows" {
```

Comment:
Do we need the if-statement as only windows reaches here?
https://github.com/aquasecurity/trivy/blob/0c056195650e38aa166c360412ebfe1b013f4510/pkg/fanal/handler/misconf/misconf_test.go#L165-L167